hail/Makefile

```diff
@@ -281,7 +281,7 @@ wheel: $(WHEEL)
 
 $(WHEEL): copy-py-files
 	# Clear the bdist build cache before building the wheel
```

Comment:
Is this comment still relevant?

Reply:
Good catch. Lemme check.

Reply:
yes, `build` does much the same so probably best to clear it. removed `bdist` comment though.